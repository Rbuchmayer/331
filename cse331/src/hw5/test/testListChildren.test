CreateGraph empty
CreateGraph oneNode
CreateGraph oneNodeOneEdge
CreateGraph twoNodesTwoEdges
CreateGraph twoNodesTwoOppositeEdges
CreateGraph complexGraph
AddNode oneNode n1
AddNode oneNodeOneEdge n2
AddEdge oneNodeOneEdge n2 n2 e1
AddNode twoNodesTwoEdges n3
AddNode twoNodesTwoEdges n4
AddEdge twoNodesTwoEdges n3 n4 e2
AddEdge twoNodesTwoEdges n3 n4 e3
AddNode twoNodesTwoOppositeEdges n5
AddNode twoNodesTwoOppositeEdges n6
AddEdge twoNodesTwoOppositeEdges n5 n6 e4
AddEdge twoNodesTwoOppositeEdges n6 n5 e5
AddNode complexGraph f
AddNode complexGraph b
AddNode complexGraph c
AddNode complexGraph d
AddNode complexGraph e
AddNode complexGraph a
AddNode complexGraph g
AddEdge complexGraph a b ab
AddEdge complexGraph f b fb
AddEdge complexGraph g f gf
AddEdge complexGraph e e ee
AddEdge complexGraph e c ec
AddEdge complexGraph b g bg
AddEdge complexGraph g c gc
AddEdge complexGraph g g gg
ListChildren oneNode n1
ListChildren oneNodeOneEdge n2
ListChildren twoNodesTwoEdges n3
ListChildren twoNodesTwoEdges n4
ListChildren twoNodesTwoOppositeEdges n5
ListChildren twoNodesTwoOppositeEdges n6
ListChildren complexGraph a
ListChildren complexGraph b
ListChildren complexGraph c
ListChildren complexGraph d
ListChildren complexGraph e
ListChildren complexGraph f
ListChildren complexGraph g