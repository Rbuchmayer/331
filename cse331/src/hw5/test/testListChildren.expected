created graph empty
created graph oneNode
created graph oneNodeOneEdge
created graph twoNodesTwoEdges
created graph twoNodesTwoOppositeEdges
created graph complexGraph
added node n1 to oneNode
added node n2 to oneNodeOneEdge
added edge e1 from n2 to n2 in oneNodeOneEdge 
added node n3 to twoNodesTwoEdges
added node n4 to twoNodesTwoEdges
added edge e2 from n3 to n4 in twoNodesTwoEdges
added edge e3 from n3 to n4 in twoNodesTwoEdges
added node n5 to twoNodesTwoOppositeEdges
added node n6 to twoNodesTwoOppositeEdges
added edge e4 from n5 to n6 in twoNodesTwoOppositeEdges
added edge e5 from n6 to n5 in twoNodesTwoOppositeEdges
added node f to complexGraph
added node b to complexGraph
added node c to complexGraph
added node d to complexGraph
added node e to complexGraph
added node a to complexGraph
added node g to complexGraph
added edge ab from a to b in complexGraph
added edge fb from f to b in complexGraph
added edge gf from g to f in complexGraph
added edge ee from e to e in complexGraph
added edge ec from e to c in complexGraph
added edge bg from b to g in complexGraph
added edge gc from g to c in complexGraph
added edge gg from g to g in complexGraph
the children of n1 in oneNode are:
the children of n2 in oneNodeOneEdge are: n2(e1)
the children of n3 in twoNodesTwoEdges are: n4(e2) n4(e3)
the children of n4 in twoNodesTwoEdges are:
the children of n5 in twoNodesTwoOppositeEdges are: n6(e4)
the children of n6 in twoNodesTwoOppositeEdges are: n5(e5)
the children of a in complexGraph are: b(ab)
the children of b in complexGraph are: g(bg)
the children of c in complexGraph are:
the children of d in complexGraph are:
the children of e in complexGraph are: c(ec) e(ee)
the children of f in complexGraph are: b(fb)
the children of g in complexGraph are: c(gc) f(gf) g(gg)